src/main/java/org/sonarlint/intellij/ui/resolve/MarkAsResolvedDialog.kt

```diff
@@ -43,12 +63,42 @@ class MarkAsResolvedDialog (
                 val status = getStatus()
 
                 if(status != null){
-                    println("go to server")
-                    println("status is:" + status.name)
+                    SonarLintUtils.getService(BackendService::class.java)
+                        .markAsResolved(BackendService.moduleId(module), issueKey, status, isTaintVulnerability)
```

Comment:
Usually we pass the module to the backend service, and we make the call to `moduleId` there, I think it's better in terms of responsibility. I know we did the same for hotspots but we should change it too.